src/LinuxCaptureService/LinuxCaptureService.cpp

```diff
@@ -228,8 +229,10 @@ grpc::Status LinuxCaptureService::Capture(
     grpc::ServerReaderWriter<CaptureResponse, CaptureRequest>* reader_writer) {
   orbit_base::SetCurrentThreadName("CSImpl::Capture");
 
-  if (grpc::Status result = InitializeCapture(reader_writer); !result.ok()) {
-    return result;
+  if (ErrorMessageOr<void> result =
+          InitializeCapture(std::make_unique<GrpcClientCaptureEventCollector>(reader_writer));
+      result.has_error()) {
+    return grpc::Status(grpc::StatusCode::ALREADY_EXISTS, result.error().message());
```

Comment:
Assuming that the resulting error is due to an already running capture is not great. Of course it's currently true, but it means we depend on some implementation detail here.

What about introducing an `enum class CaptureInitializationResult` with fields like `kSuccess` and `kAlreadyInProgress` and returning that from `InitializeCapture`?